Source/MindTheWord.js

```diff
@@ -120,14 +121,14 @@ function length(obj) {
 
 function intersect() {
   var i,
-      all,
-      shortest,
-      nShortest,
-      n,
-      len,
-      ret = [],
-      obj = {},
-      nOthers;
+    all,
```

Comment:
In this case, I actually prefer when all the vars are aligned to each other.  Does the style guide say anything specific about cases like this?